tools/functional-tester/etcd-tester/tester.go

```diff
@@ -85,21 +84,52 @@ func (tt *tester) runLoop() {
 				s.Cancel()
 			}
 
-			log.Printf("etcd-tester: [round#%d case#%d] waiting 5s for pending PUTs to be committed across cluster...", i, j)
-			time.Sleep(5 * time.Second)
+			ok := false
+			for k := 0; k < 5; k++ {
+				time.Sleep(time.Second)
+				log.Printf("etcd-tester: [round#%d case#%d.%d] checking current revisions...", i, j, k)
+				revs, err := tt.cluster.getRevision()
+				if err != nil {
+					if err := tt.cleanup(i, j); err != nil {
+						log.Printf("etcd-tester: [round#%d case#%d.%d] cleanup error: %v", i, j, k, err)
+						return
+					}
+					continue
```

Comment:
we need to print out something before the continue. probably failed to get revisions (%v), err ?